### PR TITLE
Regular payments

### PR DESCRIPTION
**Pull request by BabenkoA**

### CODE REVIEW

File: App.Common/Repositories/IRegularPaymentsRepository.cs [hunk 1]
```diff
@@ -1,6 +1,12 @@
-﻿namespace App.Repositories
+﻿using App.Models;
+using System.Collections.Generic;
+
+namespace App.Repositories
 {
     public interface IRegularPaymentsRepository
     {
+        IEnumerable<RegularPayment> GetRegularPayments();
+        void CreateRegularPayment(RegularPayment regularPayment);
```
Comment (ghost) [inline App.Common/Repositories/IRegularPaymentsRepository.cs]: Create method should returns either identifier of created entity, or the entire entity

**Review by ghost — CHANGES_REQUESTED** [review]

File: App.Common/Models/RegularPayment.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+using System.Text;
+
+namespace App.Models
+{
+    public class RegularPayment
+    {
+        public int PaymentID { get; set; }
+        public string Payer { get; set; }
+        public string Recipient { get; set; }
+        public double Payment { get; set; }
```
Comment (BabenkoA) [inline App.Common/Models/RegularPayment.cs]: Amount

File: App.RegularPayments/PaymentsManager.cs [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        }
+
+        public RegularPayment GetRegularPaymentsById(int id)
+        {
+            return _repository.GetRegularPaymentById(id);
+        }
+
+        public DateTime ShowNextPaymentData(int id)
+        {
+            return GetRegularPaymentsById(id).DateOfLastPay.AddDays(GetRegularPaymentsById(id).Period);
```
Comment (BabenkoA) [inline App.RegularPayments/PaymentsManager.cs]: make only 1 request to GetRegularPaymentsById

File: App.RegularPayments/Repositories/InMemoryRegularPaymentRepository.cs [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+        public IEnumerable<RegularPayment> GetRegularPayments()
+        {
+            return regularpayments;
+        }
+        public bool CreateRegularPayment(RegularPayment regularPayment)
+        {
+            int check = regularpayments.Count();
+            regularpayments = regularpayments.Append(regularPayment);
+            return (regularpayments.Count()) == (check + 1);
```
Comment (BabenkoA) [inline App.RegularPayments/Repositories/InMemoryRegularPaymentRepository.cs]: check if the same id exists

**Pull request merged**